packages/next-server/lib/loadable.d.ts

```diff
@@ -0,0 +1,14 @@
+/* tslint:disable */
+import React from 'react'
+
+declare namespace LoadableExport {
+  interface ILoadable {
+    <P = {}>(opts: any): React.ComponentClass<P>
+    Map<P = {}>(opts: any): React.ComponentType<P>
+    preloadAll(): Promise<any>
```

Comment:
`preloadAll` shouldn't be exposed to users if possible.

Reply:
@timneutkens Not possible 😢 , but also not necessary, users wont have access to this types unless they explicitly import from `next-server/dist/lib/loadable`